.github/workflows/audit.yaml

```diff
@@ -18,13 +18,18 @@ jobs:
       - name: install cargo audit
         run: cargo install cargo-audit
       - name: Run audit
-        # chrono, a Rust date-time crate we use for timestamp parsing was added
-        # to the RustSec vulnerability database because of a call to localtime_r.
-        # We use chrono for an extremely narrow use case of converting epoch timestamps to UTC dates
-        # (so we never invoke the problematic behavior that results from attempting to determine
-        # the local time of the current platform).
-        #
-        # There currently is no release from chrono that rectifies the issue, and we are working
-        # on a mitigation to ensure that we either use a patched version of chrono or add a deny-lint
-        # to call any chrono methods other than the method we use currently.
-        run: cargo audit --ignore RUSTSEC-2020-0071 --ignore RUSTSEC-2020-0159
+        run: cargo audit \
+          # chrono, a Rust date-time crate we use for timestamp parsing was added
+          # to the RustSec vulnerability database because of a call to localtime_r.
+          # We use chrono for an extremely narrow use case of converting epoch timestamps to UTC dates
+          # (so we never invoke the problematic behavior that results from attempting to determine
+          # the local time of the current platform).
+          #
+          # Chrono's only use is as an optional feature in `aws-smithy-types-convert`, for
+          # the narrow use-case of converting from the SDK's `DateTime` into chrono types,
+          # and we never invoke the problematic behavior that results from attempting to
+          # determine the local time of the current platform.
+          --ignore RUSTSEC-2020-0071 \
+          --ignore RUSTSEC-2020-0159 \
+          # Criterion is only used to run benchmarks during CI
```

Comment:
We should add some more detail to this to indicate serde_cbor is coming in through criterion, and perhaps a TODO comment and a tracking issue to update criterion and remove the ignore once criterion moves off of serde_cbor.

Reply:
I have updated it to link to the issue I created https://github.com/awslabs/smithy-rs/issues/1044